verify/verify-pylint.sh

```diff
@@ -20,11 +20,6 @@ set -o pipefail
 export PYLINTHOME=$(mktemp -d)
 export PATH="${TEST_SRCDIR}/pylint:${PATH}"
 
-# TODO(fejta): all python files
-pylint scenarios/*.py
-pylint jenkins/bootstrap.py
-pylint jenkins/janitor.py
-pylint jobs/*.py
-pylint queue-health/graph/graph.py
-pylint queue-health/weekly_commit_stats.py
-pylint triage/*.py
+find . -type d \( -path gubernator/third_party -o -path vendor -o -path external \) -prune -o -name *.py
```

Comment:
I think you want
`find . -type d \( -path ./gubernator/third_party/ -o -path ./vendor/ -o -path ./external/ \) -prune -o -name *.py`